manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -2677,248 +2435,6 @@ spec:
                     local:
                       inline_string: "envoy.wasm.stats"
 ---
-# Source: istiod/templates/telemetryv2_1.15.yaml
```

Comment:
we always add next release in prev releases.

Reply:
@zirain I just looked at other release branches, and they only contain the corresponding release and two prev releases. Not sure if 1.15 should be included in the 1.14 release.